pkgs/by-name/bu/buckets/package.nix

```diff
@@ -0,0 +1,66 @@
+{
+  fetchurl,
+  appimageTools,
+  lib,
+  makeWrapper,
+  stdenv,
+}:
+
+let
+
+  inherit (stdenv.hostPlatform) system;
+  throwSystem = throw "Unsupported system: ${system}";
+
+  platform =
+    {
+      x86_64-linux = "amd64";
+      aarch64-linux = "arm64";
+    }
+    .${system} or throwSystem;
```

Comment:
this should probably be done with the `meta.broken` and/or `meta.platforms` attribute to ensure the it not being called with those systems accidentally instead of throwing.